docs/tutorials/intro.md

```diff
@@ -70,102 +72,72 @@ algokit init
 
 This will launch a guided menu system to create a specific project tailored to your needs. The templates are basic starter applications for various Algorand development scenarios. To read more about templates checkout AlgoKit detailed documentation.
 
-For now we'll use the `playground` template, which is a lightweight starting point for learning and experimentation. You can initialise a project using this template by running:
+For now we'll use the `python` template, which is a lightweight starting point for learning Algorand smart contract development using Python. You can initialise a project using this template by running:
 
 ```shell
-algokit init -t playground --no-workspace
+algokit init -t python --no-workspace -a preset_name "starter"
 ```
 
 Next, you will be prompted for the name of your project. Finally, select the default value for the rest of the prompts (enter).
 
-Once finished, (if you have it installed) VS Code should automatically be opened with the initialised project and you will be prompted to install appropriate VS Code extensions. This starter app will contain one smart contract (built using the [Beaker](https://beaker.algo.xyz/) smart contract development framework) named `helloworld.py`, in the `hello_world` folder, with one method (`hello`) that takes a `String` and returns a `String`.
+Once finished, (if you have it installed) VS Code should automatically be opened with the initialised project and you will be prompted to install appropriate VS Code extensions. This starter app will contain one smart contract (built using the [Algorand Python](https://algorandfoundation.github.io/puya/) language) named `contract.py`, in the `smart_contracts/hello_world` folder, with one method (`hello`) that takes a `String` and returns a `String`. Notice that within the contract folder, there is a `deploy_config.py`, this defined the parameters for deployment of your smart contract to the select target chain (Algorand TestNet, Algorand MainNet, Algorand LocalNet or else).
 
 ![AlgoKit Playground Contract](../imgs/algokitplayground.png)
 
-## Run the Demo Application
+## AlgoKit Project structure 🏗
 
-Once the playground project is created, you will notice in the `hello_world` folder a file named `demo.py` which is a simple example of using AlgoKit to deploy and make a call to the `helloworld.py` smart contract on the LocalNet instance started earlier.
+Since you have just instantiated an official AlgoKit template, each official smart contract template includes an interactive codespace walkthrough powered by [CodeTours](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour).
```

Comment:
```suggestion
Since you have just instantiated an official AlgoKit template, each official smart contract template includes an interactive codespace walkthrough powered by [CodeTour](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.codetour).
```